server/server_test.go

```diff
@@ -1273,6 +1273,39 @@ var _ = Describe("Server package test", func() {
 		})
 	})
 
+	Describe("Substring Search ", func() {
+		BeforeEach(func() {
+			testURL("POST", networkPluralURL, adminTokenID, getNetwork("red", "red"), http.StatusCreated)
+			testURL("POST", networkPluralURL, adminTokenID, getNetwork("red1", "red"), http.StatusCreated)
+			testURL("POST", networkPluralURL, adminTokenID, getNetwork("red2", "blue"), http.StatusCreated)
+			testURL("POST", networkPluralURL, adminTokenID, getNetwork("red3", "blue"), http.StatusCreated)
+		})
+
+		AfterEach(func() {
+			testURL("DELETE", getNetworkSingularURL("red"), adminTokenID, nil, http.StatusNoContent)
+			testURL("DELETE", getNetworkSingularURL("red1"), adminTokenID, nil, http.StatusNoContent)
+			testURL("DELETE", getNetworkSingularURL("red2"), adminTokenID, nil, http.StatusNoContent)
+			testURL("DELETE", getNetworkSingularURL("red3"), adminTokenID, nil, http.StatusNoContent)
+		})
+
+		DescribeTable("match all rows by substring search", func(query string) {
```

Comment:
If you move `func(query string...` to next line, goimports/gofmt will format it a bit better. This is of course subjective.